Dockerfile

```diff
@@ -98,16 +98,28 @@ ENV CDM_SHARED_DIR=/cdm_shared_workspace
 RUN mkdir -p ${CDM_SHARED_DIR} && chmod -R 777 ${CDM_SHARED_DIR}
 RUN chown -R spark_user:spark $CDM_SHARED_DIR
 
+# Set a directory for hosting Hive metastore files - defined in config/hive-site-template.xml
+ENV HIVE_METASTORE_DIR=$CDM_SHARED_DIR/hive_metastore
+RUN mkdir -p ${HIVE_METASTORE_DIR}
+RUN chown -R spark_user:spark $HIVE_METASTORE_DIR
```

Comment:
Presumably this needs to be writable by ppl who can add deltatables?

Reply:
Yea. yea. That's right.

Reply:
So will this break that?

Reply:
I don't think so. I will handle permissions before spawner starts. Same as local spawner we did before.

Reply:
:+1: